apps/backend/src/modules/auth/auth.controller.ts

```diff
@@ -37,6 +43,35 @@ class AuthController extends BaseController {
 		});
 	}
 
+	/**
+	 * @swagger
+	 * /auth/authenticated-user:
+	 *    get:
+	 *      description: Get authenticated user's data
+	 *      responses:
+	 *        200:
+	 *          description: Successful operation
+	 *          content:
+	 *            application/json:
+	 *              schema:
+	 *                type: object
+	 *                properties:
+	 *                  message:
+	 *                    type: object
+	 *                    $ref: "#/components/schemas/User"
+	 */
+
+	private getAuthenticatedUser(
+		options: APIHandlerOptions,
+	): Promise<APIHandlerResponse> {
+		const { user } = options;
```

Comment:
But we don't have user in options, where does it come from?

Reply:
Got it. In that case we will need to wait for Serhii to finalize checking your PR